examples/remote_control_cozmo.py

```diff
@@ -32,11 +32,41 @@
     sys.exit("Cannot import from flask: Do `pip3 install --user flask` to install")
 
 try:
-    from PIL import Image
+    from PIL import Image, ImageDraw
 except ImportError:
     sys.exit("Cannot import from PIL: Do `pip3 install --user Pillow` to install")
 
 
+# Annotator for displaying RobotState (position, etc.) on top of the camera feed
+class RobotStateDisplay(cozmo.annotate.Annotator):
```

Comment:
was there a reason to make this a class, rather than just decorating a function with `@cozmo.annotate.annotator`  ?